### PR TITLE
Release v01

### PR DESCRIPTION
**Pull request by Daron976**
### In this PR:
- Set up react environment for the react group project.
- Fetched data from [REST COUNTRIES](https://restcountries.com/#api-endpoints-v3-all) and [Currency API](https://github.com/fawazahmed0/currency-api#readme).
- Built app based on fetched data.
- Cross-referenced REST COUNTRIES and Currency API to create and implement conversion functionality.
- Implemented mobile styling.
- Here is a link to the [presentation](https://www.loom.com/share/beb3de90650b47c1a9e8b90fe5c60ae9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,231 @@
...
+
+If you like this project...
+
+Feel free to give it a start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (VICTOR-CHUKWU) [inline README.md:200]: - [ ] According to project requirement kindly add an acknowledgement to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)),

File: src/modules/homepage.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        name="proceed"
+        id="proceed"
+      >
+        Proceed
+      </button>
+    </Link>
+  </section>
+);
+
+export default Homepage;
```
Comment (VICTOR-CHUKWU) [inline src/modules/homepage.js:21]: - [ ] I am puuting this comment here so you can understand what I am saying, according to the project requirement, a user should be able to search for a currency, so you will have to use a filter method in your redux to do this, kindly add this as it is the only thing left for you to finish, you have a wonderful project already :clap:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Daron976 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- [ ] Nice design
- [ ] Linters passed
- [ ] Good code base

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/homepage.js (hunk 2 above)
Comment (Daron976) [reply src/modules/homepage.js:21]: Hello 
please note that I have implemented the search bar on the third page as that is more viable

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @Daron976 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Search functionality is implemented correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**